config.schema.yaml

```diff
@@ -1,12 +1,17 @@
 ---
-imap_mailboxes:
-  description: "Mailbox parameters"
-  type: "object"
-  required: false
+smtp_accounts:
+  description: "SMTP accounts"
+  type: "array"
```

Comment:
One option would also be to make this an object and then we don't need this piece - https://github.com/StackStorm-Exchange/stackstorm-email/pull/7/files#diff-a85bc597c3be543a28e107a7b26a2b85R17, because that piece means you can't have two accounts with the same name anyway so array doesn't provide additional "value" beyond a dict / object.

Reply:
See https://github.com/StackStorm/discussions/issues/249#issuecomment-317926339 for the rationale.

Reply:
Thanks for clarification - works for me.